web/src/components/navbar/mobile/NavBarMobile.tsx

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable react/jsx-key */
+import { NavBarLogoOnly } from "@/components/navbar/NavBarLogoOnly";
+import { NavBarMobileAccountSlideOutMenu } from "@/components/navbar/mobile/NavBarMobileAccountSlideOutMenu";
+import { NavBarMobileHomeLogo } from "@/components/navbar/mobile/NavBarMobileHomeLogo";
+import { NavBarMobileQuickLinksSlideOutMenu } from "@/components/navbar/mobile/NavBarMobileQuickLinksSlideOutMenu";
+import { NavBarMobileWrapper } from "@/components/navbar/mobile/NavBarMobileWrapper";
+import {
+  AddBusinessItem,
+  GetStartedMenuItem,
+  LoginMenuItem,
+  LogoutMenuItem,
+  MyNjMenuItem,
+  ProfileMenuItem,
+  RegisterMenuItem,
+} from "@/components/navbar/shared-submenu-components";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { getNavBarBusinessTitle } from "@/lib/domain-logic/getNavBarBusinessTitle";
+import { Task } from "@/lib/types/types";
+import { ReactElement, useState } from "react";
+
+interface Props {
+  scrolled: boolean;
+  task?: Task;
+  hideMiniRoadmap?: boolean;
+  showSidebar?: boolean;
+  isLanding?: boolean;
+  previousBusinessId?: string | undefined;
+  logoOnlyType?: "NAVIGATOR_LOGO" | "NAVIGATOR_MYNJ_LOGO" | undefined;
+  currentlyOnboarding: boolean;
+  isAuthenticated: boolean;
+}
+
+export const NavBarMobile = (props: Props): ReactElement => {
+  const { business } = useUserData();
```

Comment:
dust - injecting business as a prop should turn this component into a dummy UI component